cmd/crc/cmd/daemon_windows.go

```diff
@@ -6,34 +6,38 @@ import (
 
 	"github.com/code-ready/crc/pkg/crc/api"
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/pkg/errors"
 	"golang.org/x/sys/windows/svc"
 	"golang.org/x/sys/windows/svc/debug"
 	"golang.org/x/sys/windows/svc/eventlog"
 )
 
-func runDaemon() {
+func runDaemon() error {
 	// Remove if an old socket is present
-	os.Remove(constants.DaemonSocketPath)
-	RunDaemonService("CodeReady Containers", false)
+	_ = os.Remove(constants.DaemonSocketPath)
+	return RunDaemonService("CodeReady Containers", false)
 }
 
 type daemonService struct {
 	eventLogger debug.Log
 }
 
 func (service *daemonService) Execute(args []string, r <-chan svc.ChangeRequest, changes chan<- svc.Status) (ssec bool, errno uint32) {
-	const cmdsAccepted = svc.AcceptStop | svc.AcceptShutdown
-	crcAPIServer, err := api.CreateServer(constants.DaemonSocketPath, newConfig, newMachineWithConfig)
+	apiServer, err := api.CreateServer(constants.DaemonSocketPath, newConfig, newMachineWithConfig)
 	if err != nil {
-		_ = service.eventLogger.Error(1, fmt.Sprintf("Failed to start CRC daemon service: %v", err))
+		_ = service.eventLogger.Error(1, fmt.Sprintf("Failed to start daemon service: %v", err))
 		return
 	}
-	// Windows servivce manager calls OnStart when starting the service
+	// Windows service manager calls OnStart when starting the service
 	// and it calls this method, which the service manager expects to handle
-	// commands like stop, shutdown. crcAPIServer.Serve is being called as a go routine
+	// commands like stop, shutdown. apiServer.Serve is being called as a go routine
 	// since we don't want it block
```

Comment:
Still some removal of crc/api in crcapiserver in this commit, they belong to one of the previous ones.